runtime/Go/antlr/dfa.go

```diff
@@ -5,7 +5,7 @@ import "sort"
 type DFA struct {
 	atnStartState DecisionState
 	decision      int
-	_states       map[string]*DFAState
+	states        map[string]*DFAState
```

Comment:
I'll keep it short, but I'm open to other opinions.

The tests are most important.

Idiomatic code is the cost of doing business.

Performant code doesn't matter if the tests don't pass.

We should be able to pick up on new ANTLR features and bug fixes by ensuring we run all of the tests.